### PR TITLE
Docs: Create Intro grafana

### PR DESCRIPTION
**Pull request by oddlittlebird**
Removed Add a data source from Getting started and put it in its own topic under Data sources.

Added Intro to Grafana, which is intended to be a high-level view of Grafana, the Grafana process, and Grafana features. This is part of the continuing process of improving the onboarding process for new users.

### CODE REVIEW

**Review by daniellee — CHANGES_REQUESTED** [review]
This looks really good.

I had one nitpick about the template/variable section and the broken link on the Getting Started page.

I also had another question, the new intro to Grafana seems a bit hidden to me. The Next Steps link in the Getting Started section doesn't lead anywhere - should it go to the new Intro page? The [intro to timeseries](https://grafana.com/docs/grafana/latest/guides/timeseries/) page is also hidden - the only way to find it is through searching or through the menu.

File: docs/sources/guides/intro-to-grafana.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The following sections provide an overview of things you might want to do with your Grafana database and links so you can learn more. For more guidance and ideas, check out the [Grafana Community forums](https://community.grafana.com/).
+
+## Explore metrics and logs
+
+Explore your data through ad-hoc queries and dynamic drilldown. Split view and compare different time ranges, queries and data sources side by side. Refer to [Explore]({{< relref "../features/explore/index.md" >}}) for more information.
+
+## Alerts
+
+If you're using Grafana alerting, then you can have alerts sent through a number of different [alert notifiers]({{< relref "../alerting/notifications.md" >}}), including PagerDuty, SMS, email, VictorOps, OpsGenie, or Slack.
```
Comment (marcusolsson) [inline docs/sources/guides/intro-to-grafana.md]: "Get notified about important changes in your data, by sending alerts ..."

File: docs/sources/guides/intro-to-grafana.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Configure Grafana
+
+If you're a Grafana administrator, then you'll want to thoroughly familiarize yourself with [Grafana configuration options]({{< relref "../installation/configuration.md" >}}) and the [Grafana CLI]({{< relref "../administration/cli.md" >}}).
+
+Configuration covers both config files and environment variables. You can set up default ports, logging levels, email IP addresses, security, and more.
+
+## Import dashboards and plugins
+
+Discover hundreds of [dashboards](https://grafana.com/grafana/dashboards) and [plugins](https://grafana.com/grafana/plugins) in the official library. Thanks to the passion and momentum of our community, new ones are added every week.
```
Comment (marcusolsson) [inline docs/sources/guides/intro-to-grafana.md]:
```suggestion
Discover hundreds of [dashboards](https://grafana.com/grafana/dashboards) and [plugins](https://grafana.com/grafana/plugins) in the official library. Thanks to the passion and momentum of community members, new ones are added every week.
```

File: docs/sources/guides/intro-to-grafana.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Authentication
+
+Grafana supports different authentication styles, such as LDAP and OAuth, and allows you to map users to organizations. Refer to the [User authentication overview]({{< relref "../auth/overview.md" >}}) for more information.
```
Comment (marcusolsson) [inline docs/sources/guides/intro-to-grafana.md]:
```suggestion
Grafana supports different authentication methods, such as LDAP and OAuth, and allows you to map users to organizations. Refer to the [User authentication overview]({{< relref "../auth/overview.md" >}}) for more information.
```

File: docs/sources/guides/intro-to-grafana.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+In Grafana Enterprise, you can also map users to teams: If your company has its own authentication system, Grafana allows you to map the teams in your internal systems to teams in Grafana. That way, you can automatically give people access to the dashboards designated for their teams. Refer to [Grafana Enterprise]({{< relref "../enterprise/_index.md" >}}) for more information.
+
+## Provisioning
+
+While it's easy to click, drag, and drop to create a single dashboard, power users in need of many dashboards will want to automate the setup with a script. You can script anything in Grafana. 
+
+For example, if you're spinning up a new Kubernetes cluster, you can also spin up a Grafana automatically with a script that would have the right server, IP address, and data sources preset and locked. It's also a way of getting control over a lot of dashboards.
```
Comment (marcusolsson) [inline docs/sources/guides/intro-to-grafana.md]: "Locked", as in "not possible to change"?

File: docs/sources/guides/intro-to-grafana.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+Refer to [Provisioning]({{< relref "../administration/provisioning.md" >}}) for more information.
+
+## Permissions
```
Comment (marcusolsson) [inline docs/sources/guides/intro-to-grafana.md]:
```suggestion
## Democratize metrics
```

It doesn't feel like the text reflects the heading?

**Review by marcusolsson — DISMISSED** [review]
Nice work! 👍  Added some comments. Also, some of the "Refer to" sentences are on a separate line, and some are not. I assume we want to be consistent here.

File: docs/sources/guides/intro-to-grafana.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+## Annotations
+
+Annotate graphs with rich events from different data sources. Hover over events to see the full event metadata and tags.
+
+This feature, which shows up as a graph marker in Grafana, is useful for correlating data in case something goes wrong. You can create the annotations manually—just control-click on a graph and input some text—or you can fetch data from any data source. Refer to [Annotations]({{< relref "../reference/annotations.md" >}}) for more information.
+
+## Dashboard variables
+
+[Variables]({{< relref "../reference/templating.md" >}}), also called templates, allow you to create dashboards that can be reused for lots of different use cases. Values aren't hard-coded with these templates, so for instance, if you have a production server and a test server, you can use the same dashboard for both.
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/guides/intro-to-grafana.md]: I used "variables" because while we have a topic about variables (templating.md), we don't talk much about templating in the docs. It is only mentioned in some of the data source topics.

File: docs/sources/guides/getting_started.md [hunk 7]
```diff
@@ -35,36 +35,13 @@ This step varies according to your computer operating system. Refer to the instr
 
  Congratulations, you have gotten started with Grafana! You have a dashboard and are displaying results. Feel free to experiment with what you have built, continue on to add another data source, or explore [Next steps](#next-steps).
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/guides/getting_started.md]: Fixed.

File: docs/sources/guides/intro-to-grafana.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+parent = "guides"
+weight = 200
++++
+
+# Intro to Grafana
+
+This topic provides a high-level look at Grafana, the Grafana process, and Grafana features. It is a good place to start if you want to learn more about Grafana software. To jump right in and start seeing (fake) data in graphs, refer to [Getting started]({{< relref "getting_started.md" >}}).
+
+
+Grafana is open source visualization and analytics software. It allows you to query, visualize, alert on, and explore your metrics no matter where they are stored. In plain English, it provides you with tools to turn your time-series database (TSDB) data into beautiful graphs and visualizations.
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/guides/intro-to-grafana.md]: No, this edit is not correct. You can check that by simplifying the sentence, seeing if it still works without some of the adjectives. This sentence simplifies to "Grafana is open source software."

"Grafana is an open source software" sounds kind of weird.

File: docs/sources/guides/intro-to-grafana.md (hunk 4 above)
Comment (oddlittlebird) [reply docs/sources/guides/intro-to-grafana.md]: Good question. I think I copied this content from @daniellee's article. Daniel, can you answer this question?

File: docs/sources/guides/intro-to-grafana.md (hunk 5 above)
Comment (oddlittlebird) [reply docs/sources/guides/intro-to-grafana.md]: @daniellee ?

File: docs/sources/guides/intro-to-grafana.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (daniellee) [reply docs/sources/guides/intro-to-grafana.md]: This is a good point. I would say that everything around template variables is confusing - both our UI and our docs. What's most logical for new users?

A template is a known concept but a dashboard doesn't really look like something like a Word template so maybe we should stay away from that wording. On the other hand, looking at Datadog and Chronograf they seem to have followed our lead and call them `template variables` so maybe it is too established to change now?

So in summary, to be on the safe side, I would have `[Template variables]({{< relref "../reference/templating.md" >}}) allow you...`

File: docs/sources/guides/what-is-grafana.md [hunk 9]
```diff
@@ -62,35 +56,38 @@ Configuration covers both config files and environment variables. You can set up
...
 
 While it's easy to click, drag, and drop to create a single dashboard, power users in need of many dashboards will want to automate the setup with a script. You can script anything in Grafana. 
 
-For example, if you're spinning up a new Kubernetes cluster, you can also spin up a Grafana automatically with a script that would have the right server, IP address, and data sources preset and locked. It's also a way of getting control over a lot of dashboards.
+For example, if you're spinning up a new Kubernetes cluster, you can also spin up a Grafana automatically with a script that would have the right server, IP address, and data sources preset and locked in so users cannot change them. It's also a way of getting control over a lot of dashboards.
 
 Refer to [Provisioning]({{< relref "../administration/provisioning.md" >}}) for more information.
 
-## Permissions
+## Democratize metrics
```
Comment (daniellee) [inline docs/sources/guides/what-is-grafana.md]: Should it not be `## Permissions`?

**Review by daniellee — APPROVED** [review]
Just one nitpick otherwise LGTM
